roads.mss

```diff
@@ -1048,30 +1048,32 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
 .tunnels-fill[zoom >= 10] {
 
   ::halo {
-    [feature = 'highway_motorway'][link != 'yes'][zoom >= 8][zoom < 12],
```

Comment:
This block should be kept (it is the halo for low-zoom roads). So we need both blocks now.